Makefile

```diff
@@ -20,15 +20,15 @@ deps:
 # Build steps.
 .PHONY: build
 build:
-	go build -o ${BIN} -ldflags="-s -w -X 'main.buildString=${BUILDSTR}' -X 'main.versionString=${VERSION}'" cmd/*.go
+	go build -ldflags="-s -w -X 'main.buildString=${BUILDSTR}' -X 'main.versionString=${VERSION}'" ./cmd/listmonk
```

Comment:
Let's use the `$BIN` variable here to be explicit (`-o ${BIN}`). The `make dist` also uses this.